src/main/java/de/tum/in/www1/artemis/config/migration/entries/BambooMigrationService.java

```diff
@@ -0,0 +1,551 @@
+     * Bamboo doesn't provide a REST endpoint for this action, but as we don't need to retrieve anything, it works the same way.
+     *
+     * @param buildPlanKey         The key of the build plan, which is usually the name combined with the project, e.g. 'EIST16W1-GA56HUR'.
+     * @param serverNotificationId the id of the notification to delete
+     */
+    private void deleteBuildPlanServerNotificationId(String buildPlanKey, Long serverNotificationId) {
+        MultiValueMap<String, String> parameters = new LinkedMultiValueMap<>();
+        parameters.add("buildKey", buildPlanKey);
+        parameters.add("notificationId", serverNotificationId.toString());
+
+        String requestUrl = bambooServerUrl + "/chain/admin/config/deleteChainNotification.action";
+        UriComponentsBuilder builder = UriComponentsBuilder.fromUriString(requestUrl).queryParams(parameters);
+
+        restTemplate.exchange(builder.build().toUri(), HttpMethod.POST, null, String.class);
+    }
+
+    /**
+     * Creates a new notification for the given build plan
+     * Bamboo doesn't provide a REST endpoint for this action, but as we don't need to retrieve anything, it works the same way.
+     *
+     * @param buildPlanKey          The key of the build plan, which is usually the name combined with the project, e.g. 'EIST16W1-GA56HUR'.
+     * @param serverNotificationUrl the url of the endpoint to notify
+     */
+    private void createBuildPlanServerNotification(String buildPlanKey, String serverNotificationUrl) {
+        MultiValueMap<String, String> body = new LinkedMultiValueMap<>();
+        body.add("conditionKey", "com.atlassian.bamboo.plugin.system.notifications:chainCompleted.allBuilds");
+        body.add("selectFields", "conditionKey");
+        body.add("notificationRecipientType", "de.tum.in.www1.bamboo-server:recipient.server");
+        body.add("selectFields", "notificationRecipientType");
+        body.add("webhookUrl", serverNotificationUrl);
+        body.add("buildKey", buildPlanKey);
+        MultiValueMap<String, String> headers = new LinkedMultiValueMap<>();
+        headers.add("Content-Type", "application/x-www-form-urlencoded");
+
+        String requestUrl = bambooServerUrl + "/chain/admin/config/configureChainNotification.action";
+        UriComponentsBuilder builder = UriComponentsBuilder.fromUriString(requestUrl);
+
+        HttpEntity<MultiValueMap<String, String>> request = new HttpEntity<>(body, headers);
+
+        restTemplate.exchange(builder.build().toUri(), HttpMethod.POST, request, String.class);
+    }
+
+    /**
+     * Deletes the given trigger id for the given build plan
+     * Bamboo doesn't provide a REST endpoint for this action, but as we don't need to retrieve anything, it works the same way.
+     *
+     * @param buildPlanKey The key of the build plan, which is usually the name combined with the project, e.g. 'EIST16W1-GA56HUR'.
+     * @param triggerId    the id of the notification to delete
```

Comment:
Is trigger = notification? 🤔 The comment seems to use both as synonyms

Reply:
No, they are not the same; that's a typo. Thanks!